components/organisms/Header.js

```diff
@@ -1,33 +1,13 @@
 import PropTypes from "prop-types";
 
-import { SearchBar } from "../atoms/SearchBar";
-import { useState } from "react";
-
 export function Header(props) {
-  //State for Search input
-  const [searchText, setSearchText] = useState(null);
-
-  //Handler for search input
-  function handleInputChange(e) {
-    setSearchText(e.target.value);
-  }
-
   return (
-    <div className="w-full flex  h-auto flex-col mt-8 lg:flex lg:flex-row  xl:flex xl:flex-row   justify-between ">
-      <div className="mb-4">
```

Comment:
whats the point of this div ? couldn't the margin just be added to the image ?

Reply:
The header is updated with just a img .